_packages/napari-nibabel.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: package
+name: "napari-nibabel"
+tag: "napari-nibabel"
```

Comment:
Tags haven't really been used correctly, but there are specific ones that were intended for packages here https://github.com/nipy/nipy.github.com/blob/94eb4356705ad764b8ba1128cdb2e14e702f2e2a/_data/package-categories.yml (and you could add another category if needed). Repeating the name of the package isn't super meaningful.